### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Rayan84**
This project is a collaborative effort to achieve replica of Newsweek magazine https://www.newsweek.com/.
Technologies Used:

- HTML
- CSS
- Bootstrap
- JavaScript

### CODE REVIEW

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

Great job so far with this project, team! 👍 
And with only using 141 lines of CSS, amazing! 👏 
Pretty impressive your use of JS in these project! ⭐ 
I can see you really know how to use bootstrap, and very well!!! 💯 
Congratulations!!! 🎂 

However, you need to do some changes before the project gets the approval! but seeing how you use bootstrap it will not take you too long!
Please, check my comments below 👇 
And once the required changes are complete resubmit your code for a new code review

Good luck with the changes, team, and keep the good work up 😃 

You can reach me out in Slack if you have any question or doubt.

## CHANGES

-General:
   - In features and top story the topic must be included inside the image, same in the magazine and editors pick:
Capture1
Capture
   - One small section is missing and it is important for smaller screens
Capture2
   - In the magazine section, the first image is different from the rest. Kindly add it
Capture3
   - My turn section and culture& travel sections are also missing, please add it to your project
  - In the opinion section many opinions are missing
  - In the original website the first breakdown is at 1000px, your is at 1200px. Please, keep the breakdowns as in the original site.
   - The navbar should fit the 100% of the screen always. Please fix it.
Capture5
Capture4
   - Please fix the images in the latest sections, they are overlapping
- Breakpoint 996px
   - Your navbar does not let to see the title of the top story and features stories sections, please fix it.
Capture6
   - The in the magazine and the editors pick sections are not following the layout of the original page, please fix it
Capture8
Capture7
   - Us, culture, sports, world, business, tech and science sections are not following the original layout neither. Please modify it
Capture10
Capture9
   - Please adjust the size of the columns in the newsletter section
Capture12
Capture11
- Breakpoint 768px
  - The order of the sections should be: top story, featured stories, opinion, more stories, my turn, culture and travel. Kindly make yours in the same order.
   - Please fix the width of the world, business, sport, tech&science.... sections, also the layout must be fixed.
Capture14
Capture13

   - The image in the newsletter section should disappear, please remove your for this screen size.
- Please also fix the margin of the section for the small devices, adjust it to the original page.

File: index.html [hunk 1]
```diff
@@ -15,11 +15,1765 @@
...
+
+          </div>
+          <div class="col-lg-4">
+            <h4 class="text-red font-weight-bold">NEWSLETTER</h4>
+            <div class="row border m-1">
+              <div class="col col-md-3 mr-1 d-sm-none d-md-block newsletter-img">
+                <span class="newsletter-img"></span>
+              </div>
+              <article class="col col-md-7 col-sm-10 m-auto py-3">
+                <h5>The Starting 5</h5>
```
Comment (umairarshadbutt) [inline index.html]: Align center this heading

File: assets/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  .max991-display-none {
+    display: none;
+  }
+
+  main {
+    padding-top: 50px;
+  }
+
+  .footer-social {
+    text-align: left !important;
```
Comment (umairarshadbutt) [inline assets/css/styles.css]: Remove `!imortant`

File: index.html [hunk 3]
```diff
@@ -15,11 +15,1765 @@
...
+                      The U.S. Needs a Cyber Strategy Designed for Defense
+                    </a>
+                  </h6>
+                  <span class="text-uppercase text-black-50">
+                    BY <a class="text-black-50" href="#"> BONNIE KRISTIAN </a>
+                  </span>
+                </div>
+                <div class="col-4">
+                  <img
+                    style="width: 4rem"
```
Comment (umairarshadbutt) [inline index.html]: Remove inline CSS

File: index.html [hunk 4]
```diff
@@ -15,11 +15,1765 @@
...
+                  </h6>
+                  <span class="text-uppercase text-black-50">
+                    BY <a class="text-black-50" href="#"> ROBERT REICH</a>
+                  </span>
+                </div>
+              </li>
+
+              <li class="media py-3 border-bottom">
+                <img
+                  style="width: 4rem"
```
Comment (umairarshadbutt) [inline index.html]: Remove inline CSS

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

File: index.html [hunk 5]
```diff
@@ -15,11 +15,1765 @@
...
+                  </h6>
+                  <span class="text-uppercase text-black-50">
+                    BY <a class="text-black-50" href="#"> KELLY SHACKELFORD</a>
+                  </span>
+                </div>
+              </li>
+
+              <li class="media py-3 border-bottom">
+                <img
+                  style="width: 4rem"
```
Comment (umairarshadbutt) [inline index.html]: Remove inline CSS

File: index.html [hunk 6]
```diff
@@ -15,11 +15,1765 @@
...
+                  </h6>
+                  <span class="text-uppercase text-black-50">
+                    BY <a class="text-black-50" href="#">  Alan Dershowitz</a>
+                  </span>
+                </div>
+              </li>
+
+              <li class="media py-3 border-bottom">
+                <img
+                  style="width: 4rem"
```
Comment (umairarshadbutt) [inline index.html]: Remove inline CSS

File: index.html [hunk 7]
```diff
@@ -15,11 +15,1765 @@
...
+              class="mt-3 mb-2 text-uppercase text-black-50 font-weight-bolder"
+            >
+              THE DEBATE
+            </h6>
+
+            <!-- Opinion Card list  -->
+            <ul class="list-unstyled">
+              <li class="media py-3 border-bottom border-bottom-double">
+                <img
+                  style="width: 4rem"
```
Comment (umairarshadbutt) [inline index.html]: Remove inline CSS

File: index.html [hunk 8]
```diff
@@ -15,11 +15,1765 @@
...
+                    alt="..."
+                  />
+                </div>
+              </li>
+
+              <!-- Opinion Section -->
+              <li class="pt-3"><strong>Opinion</strong></li>
+              <li class="media py-3 border-bottom">
+                <img
+                  style="width: 4rem"
```
Comment (umairarshadbutt) [inline index.html]: Remove inline CSS

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi 
Good job with the changes! :tada: We are almost there.
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly make sure the Last reviewer's changes and work on the suggested changes 👇.
- [x] Add heading of **My turn** section and **culture& travel** sections 
- [x] Please adjust the size of the columns in the newsletter section
- [x] Remove inline CSS and `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)




##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Rayan84 and @Lichapa
**Good job with the changes! :tada: We are almost there.**

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.

- [ ] Manage the sections order like the original landing page on small screens

### Navbar
- [x] There is a still issue in the navbar on medium and small screens

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://lichapa.github.io/Newsweek-clone/)|
|:---:|:---:|
|source-page|clone|

### Subscribe
- [x] Text should be overlapped on the medium and small screen

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://lichapa.github.io/Newsweek-clone/)|
|:---:|:---:|
|source-page|clone|

### Footer
- [x] remove `mt` from social media icons (on small screens)

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://lichapa.github.io/Newsweek-clone/)|
|:---:|:---:|
|source-page|clone|


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by arslanbisharat — APPROVED** [review]
### Congratulations APPROVED :clap:
Hello, I can see you worked really hard on this project. and I'm very happy to approve this project with the latest changes you made in this project. :satisfied:
_The clone shows a good understanding and usage of HTML CSS💯_

- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**